cmd/agent/app/agent.go

```diff
@@ -36,6 +36,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/cache"
 	"sigs.k8s.io/controller-runtime/pkg/config"
 	"sigs.k8s.io/controller-runtime/pkg/healthz"
+	"sigs.k8s.io/controller-runtime/pkg/log"
```

Comment:
Can you remind me why we need to use the logger from controller-runtime?

Any chance to keep using the klog?